tests/run-pass/ints.rs

```diff
@@ -1,34 +1,25 @@
-#![feature(custom_attribute)]
```

Comment:
I noticed more files still have this stuff. I'll change all the rest, and leave `ints.rs` so we don't clash.